private/defs.bzl

```diff
@@ -214,7 +214,6 @@ def cc_launcher(ctx, cc_toolchain, src, deps):
         compilation_outputs = objs,
         linking_contexts = [info.linking_context for info in infos],
         user_link_flags = defaults.linkopts,
-        grep_includes = ctx.executable._grep_includes,
```

Comment:
could you also remove the `_grep_includes` attributes from all rule definitions?

Reply:
Usually I was splitting removal of usages from the removal of the attribute itself, but in this case it is safe to remove them together. Done.